scripts/race-test.sh

```diff
@@ -0,0 +1 @@
+go test -race ./...
```

Comment:
I am of two opinions here: 1) We add this into build.sh and pass arguments into build.sh to trigger running this (and other future things). 2) We rename race-test.sh to test.sh and move all tests into it, again with arguments for this test. Thoughts?

Reply:
I think tests should be a part of build. There is no reason to waste upstream resources if it doesn't pass tests locally. Tests are fast at this point. If they ever get too slow, we can reconsider.